package.json

```diff
@@ -8,7 +8,7 @@
     "firebase": "^7.21.1",
     "geofirestore": "^4.3.0",
     "geolib": "^3.3.1",
-    "leaflet": "^1.7.1",
+    "leaflet": "1.6.0",
```

Comment:
@oori Why was this required?

Reply:
@uriklar Leaflet 1.7 has an issue Safari (on all platforms), it sends duplicate events (like `onMoveEnd`), which makes mess. See my comment above (`debounce`, etc..), also note it's unrelated to my PR, it exists on online site and sometimes crashes Safari randomly on map clicks (_react state changes timing_), you just didn't get enough complaints...   There are a couple of open issues on leaflet, see: https://github.com/Leaflet/Leaflet/issues/7255,  reverting to leaflet 1.6 is the safest solution for now.